modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -453,9 +453,11 @@ def treatment_centers
 
       def get_treatments
         @auto_claim['treatments'].map do |tx|
-          center = "#{tx['center']['name']}, #{tx.dig('center', 'city')}, #{tx.dig('center', 'state')}"
+          if tx['center'].present?
+            center = "#{tx.dig('center', 'name')}, #{tx.dig('center', 'city')}, #{tx.dig('center', 'state')}"
+          end
           name = tx['treatedDisabilityNames'].join(', ')
-          tx['treatmentDetails'] = "#{name} - #{center}"
+          tx['treatmentDetails'] = center.blank? ? name : "#{name} - #{center}"
```

Comment:
few questions with this one. In the event that a name, city or state is null, we'll just have a gap and ` ,`. We also have the issue with the `.join` throwing an error. 

My thought is to swap these few lines to something like below.

```
          center_data = tx.dig('center')
          center = ''
          if center_data.present?
            center_name = center_data.transform_keys(&:to_sym).values_at(:name, :city, :state).compact.reject{ |a| a.blank? }
            center = center_name.join(', ')
          end
          name = tx.dig('treatedDisabilityNames')&.join(', ')
          tx['treatmentDetails'] = "#{name}#{(name.present? && center.present?) ? ' - ' : ''}#{center}"
```
seems overly complex but has meaning. we can check center data to not be nil.. set center to an empty string so if `center = center_name.join(', ')` doesn't run, and then `.compact` to remove nil and `.reject` section to remove empty strings sent by a consumer to remove those. That specific `.join(', ')` should never fail because then we would not have any data so `center_data.present?` would fail. 

For example, if we send the below, you can see the empty values, comma separated. 
```
{
                    "treatedDisabilityNames": ["hearing loss"],
                    "center": {
                        "name": "",
                        "city": "AL"
                    }
                }

"hearing loss - , AL, "
```

Probably a more efficient way to do it, but in testing we should end up with stuff like the below....

```
{
                    "treatedDisabilityNames": ["hearing loss"],
                    "center": {
                        "name": "Hospita",
                        "city": "New York City",
                        "state": "AL"
                    }
                }

"hearing loss - Hospita, New York City, AL"


{
                    "treatedDisabilityNames": ["hearing loss"],
                    "center": {
                        "name": "",
                        "state": "AL"
                    }
                }
                
"hearing loss - AL"


{
                    "center": {
                        "name": "",
                        "city": "New York"
                    }
                }
                
"New York"
```